_posts/2025-05-20-firefox-security-privacy-newsletter-2025-q1.md

```diff
@@ -0,0 +1,72 @@
+
+**Away from DNT and towards GPC**: In Q1, Firefox became the first major browser to remove the “Do Not Track” checkbox from the privacy preferences/settings, recognising that Global Privacy Control (GPC) is a better option for 2025\. See [SUMO article](https://support.mozilla.org/en-US/kb/how-do-i-turn-do-not-track-feature) for details.
+
+## Core Security
+
+**Hardening Firefox:** Starting in Firefox 135, Firefox UI uses a strict Content-Security-Policy (CSP) in [browser.xhtml](https://searchfox.org/mozilla-central/source/browser/base/content/browser.xhtml) to prevent XSS-like sandbox escapes ([Bug 1935985](https://bugzilla.mozilla.org/show_bug.cgi?id=1935985)). This involved modifying over 600 inline event handlers and a careful rollout with notice to people who run modified Firefox configurations. A detailed description of these security hardening efforts are available in our blogpost: [Hardening the Firefox Frontend with Content Security Policies](https://attackanddefense.dev/2025/04/09/hardening-the-firefox-frontend-with-content-security-policies.html).
+
+**Sandboxing:** The Windows sandbox for Content Processes has been [moved to USER\_RESTRICTED](https://bugzilla.mozilla.org/show_bug.cgi?id=1403931) and rolled out in Firefox 134 and is enabled by default in Firefox 135\. This change removes read access from most of the filesystem and registry for the content process. Before, read access was generally allowed outside of the User’s home directory, which has been blocked since Firefox 56\. In Firefox 138, by [further differentiating the Gecko Media Plugin process sandbox policy](https://bugzilla.mozilla.org/show_bug.cgi?id=1952926), we were also able to enable win32k lockdown and Arbitrary Code Guard (blocking dynamic code) for the openh264 GMP process.
+
+**CSS Fuzzing:** We are also attributing a major piece of our platform goals to [Interop 2025](https://wpt.fyi/interop-2025): Enhancements in CSS syntax generation have shown immediate results and will continue to support the work throughout the year.
+
+**XSLT Fuzzing**: Early in Q1, we have also received an experimental XSLT fuzzer from Ivan Fratric of Google Project Zero that is now running continuously in our lab. Thank you, Ivan\!
+
+**NSS Fuzzing:** Our fuzzing focus on NSS (Network Security Services) has also shown a significant uptick in NSS fuzzing coverage across both oss-fuzz as well as our internal tools. We also added a TSan target for NSS, ensuring stricter thread-safety going forward.
```

Comment:
I think it would be nice to say coverage has increased by 13% or whatever the value was.